In [11]:
import pandas as pd
import numpy as np

**To get a player's info**

In [19]:
# To get a player's info
from re import search as research
df_pi = pd.read_csv('data/players_info.csv')
# df_pi.head()
df_pi[df_pi['player_name'].str.contains('gavas',case=False)]

player_id  player_object_id     player_name         dob  dod gender  \
687        1378             28794  Sunil Gavaskar  1949-07-10  NaN      M   
5658       7133             28792  Rohan Gavaskar  1976-02-20  NaN      M   

       batting_style           bowling_style  country_id  \
687   right-hand bat        right-arm medium         6.0   
5658   left-hand bat  slow left-arm orthodox         6.0   

                                              image_url  \
687   https://img1.hscicdn.com/image/upload/f_auto,t...   
5658  https://img1.hscicdn.com/image/upload/f_auto,t...   

                                         image_metadata  
687   {'id': 246933, 'url': '/db/PICTURES/CMS/246900...  
5658  {'id': 302578, 'url': '/db/PICTURES/CMS/302500...

**Using the test batting card**

In [20]:
df = pd.read_csv('data/test_batting_card.csv')
df.head()

Match ID  innings        team  batsman  runs  balls  fours  sixes  \
0   1213062        1  Bangladesh    48391   3.0    5.0    0.0    0.0   
1   1213062        1  Bangladesh    80555   0.0    2.0    0.0    0.0   
2   1213062        1  Bangladesh    70872  44.0  110.0    6.0    0.0   
3   1213062        1  Bangladesh    58225  30.0   59.0    5.0    0.0   
4   1213062        1  Bangladesh    47352  25.0   48.0    4.0    0.0   

   strikeRate  isOut wicketType   fielders   bowler  
0       60.00   True        lbw        NaN  59220.0  
1        0.00   True     caught  ['53493']  95855.0  
2       40.00   True     caught  ['54018']  59220.0  
3       50.84   True     caught  ['54018']  95855.0  
4       52.08   True     caught  ['53493']  95855.0

**Let's look at some simple stats for Sunil Gavaskar - player_id=1378**